CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## UNRELEASED
 
+BREAKING CHANGES:
+* Helm
+  * API Gateway: Re-use connectInject.consulNamespaces instead of requiring that apiGateway.consulNamespaces have the same value when ACLs are enabled. [[GH-1169](https://github.com/hashicorp/consul-k8s/pull/1169)]
```

Comment:
Not a big deal but for BC's it's nice to tell the user what to do. So in this case if their `apiGateway.consulNamespaces` is the same as their `connectInject.consulNamespaces` configuration, they don't have to do anything. If it's different, then they need to be aware that the `connectInject.consulNamespaces` will now be used.